### PR TITLE
Feature/expandable

### PR DESCRIPTION
**Pull request by iznv**

### CODE REVIEW

File: Sources/ExpandableCellHeightCalculator.swift [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import UIKit
+
+public final class ExpandableCellHeightCalculator: RowHeightCalculator {
+
+    private(set) weak var tableView: UITableView?
```
Comment (petropavel13) [inline Sources/ExpandableCellHeightCalculator.swift]: just private?

File: Sources/UITableViewCell+Extensions.swift [hunk 2]
```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+extension UITableViewCell {
+
+    public var tableView: UITableView? {
```
Comment (petropavel13) [inline Sources/UITableViewCell+Extensions.swift]: remove public for all methods in file

**Review by petropavel13 — CHANGES_REQUESTED** [review]

File: Sources/ExpandableCellHeightCalculator.swift (hunk 1 above)
Comment (iznv) [reply Sources/ExpandableCellHeightCalculator.swift]: ok

File: Sources/UITableViewCell+Extensions.swift (hunk 2 above)
Comment (iznv) [reply Sources/UITableViewCell+Extensions.swift]: discussed

**Review by petropavel13 — APPROVED** [review]

**Pull request merged**